react/typings/graphql.d.ts

```diff
@@ -1,3 +1,11 @@
+/* eslint-disable import/order */
+import { RenderContext } from 'vtex.render-runtime'
+
+declare global {
+  // eslint-disable-next-line no-redeclare
+  const __RUNTIME__: RenderContext
+}
+
```

Comment:
move this to the file `react/typings/global.d.ts`

Reply:
Sorry, I confused the names. I will use a commit --amend